content/packages/working-with-a-github-packages-registry/working-with-the-npm-registry.md

```diff
@@ -250,5 +250,5 @@ If your instance has subdomain isolation disabled:
 
 ## Using the official npm registry
 
-{% data variables.product.prodname_registry %} allows you to access the official npm registry at `registry.npmjs.com`, if your {% data variables.product.prodname_ghe_server %} administrator has enabled this feature. For more information, see [Connecting to the official npm registry](/admin/packages/configuring-package-ecosystem-support-for-your-enterprise#connecting-to-the-official-npm-registry).
+{% data variables.product.prodname_registry %} allows you to access the official npm registry at `registry.npmjs.com`, if your {% data variables.product.prodname_ghe_server %} administrator has enabled this feature. For more information, see [Connecting to the official `npm registry`](/admin/packages/configuring-package-ecosystem-support-for-your-enterprise#connecting-to-the-official-npm-registry).
```

Comment:
We wouldn't accept this change as "npm registry" is how it's styled in the [docs that are being linked](https://docs.github.com/en/enterprise-server@3.11/admin/packages/configuring-package-ecosystem-support-for-your-enterprise#connecting-to-the-official-npm-registry), and in the [npm docs](https://docs.npmjs.com/).